src/hardware/gus.cpp

```diff
@@ -1498,7 +1517,7 @@ void init_gus_dosbox_settings(Section_prop &secprop)
 void GUS_AddConfigSection(Config *conf)
 {
 	assert(conf);
-	Section_prop *sec = conf->AddSection_prop("gus", &gus_init);
+	Section_prop *sec = conf->AddSection_prop("gus", &gus_init, true);
```

Comment:
Please put a TODO (maybe here, maybe some other place will be more appropriate) about setting up `ULTRASND` variable.

It should be possible to fix it, so the variable would be updated dynamically (maybe via routing through SET command, maybe directly).

Reply:
Will add the TODO; good suggestion.

Yes, it turns out this was a lot more work than I anticipated.  First, I checked if there was an "Uninstall()" or "Remove()" type call for `autoexec` objects, but there isn't.

I also checked if I could apply a new `Autoexec` entry called `SET ULTRASND=`, which can manually be used to unset the variable, like you mentioned.  But it seems `Autoexec` objects are only effective during the startup phase.. the code compiled and ran, but had no effect.

If you know how to programmatically call `SET`, then that should do it!